source/tyr/migrations/versions/361f3993960e_import_stops_in_global_autocomplete_.py

```diff
@@ -0,0 +1,24 @@
+"""import stops in global autocomplete (mimir)
+
+Revision ID: 361f3993960e
+Revises: 57ee5581cc30
+Create Date: 2016-12-23 11:48:47.003811
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '361f3993960e'
+down_revision = '57ee5581cc30'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('instance', sa.Column('import_stops_in_mimir', sa.Boolean(), nullable=True))
+    op.execute('update instance set import_stops_in_mimir=false')
```

Comment:
is this supposed to set the default value in db? have you tried `op.add_column(..., server_default=xx, ...)`?